{{ cookiecutter.formal_name }}/Dockerfile

```diff
@@ -29,3 +29,10 @@ RUN apt-get update -y && \
 RUN python${PY_VERSION} -m ensurepip
 RUN python${PY_VERSION} -m pip install --upgrade pip
 RUN python${PY_VERSION} -m pip install --upgrade setuptools
+
+# Ensure Docker user matches host user (beeware/briefcase#403)
```

Comment:
Worth clarifying that "matches" means the UID and GID match, not the username and groupname.